core/src/main/java/tc/oc/pgm/core/CoreModule.java

```diff
@@ -118,13 +119,14 @@ public CoreModule parse(MapFactory context, Logger logger, Document doc)
         }
 
         String modes = coreEl.getAttributeValue("modes");
-        List<String> modeList = new ArrayList<>();
+        List<Mode> modeList = new ArrayList<>();
```

Comment:
Make this an ImmutableSet<Mode>, like the example in https://github.com/PGMDev/PGM/blob/dev/core/src/main/java/tc/oc/pgm/flag/FlagParser.java#L105